--- gnustep-base-1.9.1/SSL/GNUmakefile.orig	2004-01-13 03:39:43.000000000 +0100
+++ gnustep-base-1.9.1/SSL/GNUmakefile	2004-03-03 00:45:20.064286240 +0100
@@ -25,13 +25,13 @@
 # Install into the system root by default
 GNUSTEP_INSTALLATION_DIR = $(GNUSTEP_SYSTEM_ROOT)
 
--include ../config.mak
--include config.mak
-
 GNUSTEP_LOCAL_ADDITIONAL_MAKEFILES=../base.make
 
 include $(GNUSTEP_MAKEFILES)/common.make
 
+-include ../config.mak
+-include config.mak
+
 srcdir = .
 PACKAGE_NAME = gnustep-base
 
