--- gnustep-base-1.6.0/Source/Makefile.preamble.orig	2003-03-15 05:47:55.000000000 +0100
+++ gnustep-base-1.6.0/Source/Makefile.preamble	2003-05-14 21:03:55.000000000 +0200
@@ -78,7 +78,7 @@
 libgnustep-base_LIBRARIES_DEPEND_UPON += -lobjc
 endif
 ifeq ($(shared),yes)
-libgnustep-base_LIBRARIES_DEPEND_UPON += $(CONFIG_SYSTEM_LIBS) 
+libgnustep-base_LIBRARIES_DEPEND_UPON += -lobjc -ldl $(CONFIG_SYSTEM_LIBS) 
 endif
 
 #
