--- gnustep-base-1.15.3/config/pathtls.m4.orig	2008-01-05 14:09:39.000000000 +0000
+++ gnustep-base-1.15.3/config/pathtls.m4	2008-10-08 19:03:13.000000000 +0000
@@ -43,7 +43,7 @@
 int
 main()
 {
-  system("touch conf.tlstest");
+  if (system("touch conf.tlstest")) return 1;
 
   if (gnutls_check_version("$min_tls_version") == 0)
     {
@@ -91,7 +89,7 @@
 int
 main()
 {
-  system("touch conf.tlstest");
+  if (system("touch conf.tlstest")) return 1;
 
   if (gnutls_check_version("$min_tls_version") == 0)
     {
--- gnustep-base-1.15.3/config/pathxml.m4.orig	2006-02-11 18:22:57.000000000 +0000
+++ gnustep-base-1.15.3/config/pathxml.m4	2008-10-08 19:03:43.000000000 +0000
@@ -56,7 +56,7 @@
   int major, minor, micro;
   char *tmp_version;
 
-  system("touch conf.xmltest");
+  if (system("touch conf.xmltest")) return 1;
 
   tmp_version = xmlStrdup("$min_xml_version");
   if(sscanf(tmp_version, "%d.%d.%d", &major, &minor, &micro) != 3) {
@@ -133,7 +131,7 @@
   int major, minor, micro;
   char *tmp_version;
 
-  system("touch conf.xmltest");
+  if (system("touch conf.xmltest")) return 1;
 
   tmp_version = xmlStrdup("$min_xml_version");
   if(sscanf(tmp_version, "%d.%d.%d", &major, &minor, &micro) != 3) {
