--- gstep-0.6.0/base/Source/GNUmakefile.unicode	Tue Sep  7 22:58:44 1999
+++ gstep-0.6.0/base/Source/GNUmakefile	Sun Sep 19 01:11:24 1999
@@ -442,7 +442,8 @@
 unicode/nextstep.h
 
 HEADERS_INSTALL = $(GNU_HEADERS) \
-                  $(BASE_HEADERS)
+                  $(BASE_HEADERS) \
+                  $(UNICODE_HEADERS)
 
 GENERATED_HFILES = \
 dynamic-load.h \
--- gstep-0.6.0/base/Source/Makefile.postamble.unicode	Tue Sep  7 22:58:46 1999
+++ gstep-0.6.0/base/Source/Makefile.postamble	Sun Sep 19 01:29:21 1999
@@ -53,6 +53,10 @@
 	  $(INSTALL_DATA) ../Headers/gnustep/base/$$file \
 	    $(GNUSTEP_HEADERS)/gnustep/base/$$file ; \
 	done
+	for file in $(UNICODE_HEADERS); do \
+	  $(INSTALL_DATA) ../Headers/gnustep/$$file \
+	    $(GNUSTEP_HEADERS)/gnustep/$$file ; \
+	done
 	$(INSTALL_DATA) $(GNUSTEP_TARGET_CPU)/$(GNUSTEP_TARGET_OS)/GSConfig.h \
 	    $(GNUSTEP_HEADERS)/$(GNUSTEP_TARGET_CPU)/$(GNUSTEP_TARGET_OS)
 	$(INSTALL_DATA) $(GNUSTEP_TARGET_CPU)/$(GNUSTEP_TARGET_OS)/mframe.h \
@@ -83,6 +87,9 @@
 after-uninstall::
 	for file in $(GNU_HEADERS); do \
 	  rm $(GNUSTEP_HEADERS)/gnustep/base/$$file ; \
+	done
+	for file in $(UNICODE_HEADERS); do \
+	  rm $(GNUSTEP_HEADERS)/gnustep/$$file ; \
 	done
 
 # Things to do before cleaning
